### ISSUE TITLE
Pytorch Implementation 

### ISSUE THREAD

**Issue by Andrew-Brown1** [XinshaoAmosWang/Ranked-List-Loss-for-DML#6]
Hi, I was wondering if you had a Pytorch implementation of this great piece of work?

**Comment by XinshaoAmosWang** [comment]
Hi @Andrew-Brown1 , thanks so much for your interest. 

I am thinking about finding some time to reimplement our papers in Pytorch or Tensorflow. It may take some time, I will let you know when they are ready. 

By the way, if you are interested in deep metric learning, we have some other simple and effective methods that you may be interested in.  
1. https://www.reddit.com/r/MachineLearning/comments/f4x1sh/r_instance_cross_entropy_for_deep_metric_learning/

2. https://github.com/ppriyank/-Online-Soft-Mining-and-Class-Aware-Attention-Pytorch/issues/4

Thanks.

**Comment by XinshaoAmosWang** [comment]
Hi @Andrew-Brown1 , 

I am excited to find a third-party reimplementation, though I have not read the implementation yet 
https://github.com/michuanhaohao/reid-strong-baseline
https://github.com/Qidian213/Ranked_Person_ReID

RLL is applied for person re-identification (person image/video retrieval problem) and it works well.

**Comment by say2sarwar** [comment]
There RLL can't reproduce the results in papers. can any one help me out?

**Comment by XinshaoAmosWang** [comment]
Hi @say2sarwar, thanks for your interest.  
For official code, we only have caffe version. Its matlab interface is easy to use. 
https://github.com/XinshaoAmosWang/Ranked-List-Loss-for-DML/issues/7
